v2/src/components/appInfoForm/index.tsx

```diff
@@ -237,24 +237,41 @@ export default class AppInfoForm extends React.PureComponent<PropsWithChildren<P
         </span>
     )
 
+    getCurrentFormValuesAsString = () => {
```

Comment:
What is this function for?

Reply:
This function is used to generate a string which we display inside the snippet of `AppInfoForm` values ( as shown in below images ). It checks for the fields that the current AppInfoForm instance handles and only returns their values.

<img width="702" alt="Screenshot 2022-03-18 at 12 50 59 PM" src="https://user-images.githubusercontent.com/92283713/158955101-2738869c-d150-4d01-895d-b8b4b48e5765.png">

<img width="701" alt="Screenshot 2022-03-18 at 12 51 09 PM" src="https://user-images.githubusercontent.com/92283713/158955123-5bdcb0fd-8aa5-40e8-819a-19bac3d2ad8c.png">